### PR TITLE
Style Guide and Diataxis Review - Hornet I

### PR DESCRIPTION
**Pull request by charlesthompson3**
<!---
# Description

Updated documentation in line with TW style guide and diataxis. Sections include:

- Getting started
- Nodes 101
- Security 101 

Fixes #(issue)

## Type of change

Please delete options that are not relevant.

- [x] Documentation Fix

# Checklist:
- [x] I have made corresponding changes to the documentation
- [x] I have selected the `develop` branch as the target branch

### CODE REVIEW

**Comment by charlesthompson3** [comment]
@lucas-tortora @aleksei-korolev

File: documentation/docs/getting_started/getting_started.md [hunk 1]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
 
 ### Linux (and Raspberry Pi)
 
-Recommended installation methods:
+Hornet on Linux can be installed using:
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/getting_started.md]:
```suggestion
You can install Hornet on Linux using:
```

File: documentation/docs/getting_started/getting_started.md [hunk 2]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
 
-- [Hornet apt repository](hornet_apt_repository.md#hornet-apt-repository-linux-distro-specific)
-- [Docker image](using_docker.md)
+- [The hornet apt repository](https://wiki.iota.org/hornet/getting_started/hornet_apt_repository).
+- [The docker image](https://wiki.iota.org/hornet/getting_started/using_docker).
 
-Other installation methods:
+It can also be installed using:
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/getting_started.md]:
```suggestion
Alternatively, you can build Hornet by following these instructions:
```

File: documentation/docs/getting_started/getting_started.md [hunk 3]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
 
-- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries)
-- [Build from source](hornet_apt_repository.md#build-from-source)
+- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries), or
+- [Built from the source](hornet_apt_repository.md#build-from-source).
 
 ### MacOS
 
-Recommended installation methods:
+Hornet on MacOS can be installed using:
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/getting_started.md]:
```suggestion
You can install Hornet on MacOS using:
```

File: documentation/docs/getting_started/getting_started.md [hunk 4]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
 
-- [Docker image](using_docker.md)
+- [The docker image](https://wiki.iota.org/hornet/getting_started/using_docker).
 
-Other installation methods:
+It can also be installed using:
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/getting_started.md]:
```suggestion
Alternatively, you can build Hornet by following these instructions:
```

File: documentation/docs/getting_started/getting_started.md [hunk 5]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
 
-- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries)
+- [Prebuilt binary files](https://wiki.iota.org/hornet/getting_started/using_docker#starting-an-existing-hornet).
 
 ### Windows
 
-Recommended installation methods:
+Hornet on Windows can be installed using:
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/getting_started.md]:
```suggestion
You can install Hornet on windows using:
```

File: documentation/docs/getting_started/getting_started.md [hunk 6]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
 
-- [Docker image](using_docker.md)
+- [The docker image](https://wiki.iota.org/hornet/getting_started/using_docker).
 
-Other installation methods:
+It can also be installed using:
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/getting_started.md]:
```suggestion
Alternatively, you can build Hornet by following these instructions:
```

File: documentation/docs/getting_started/getting_started.md [hunk 7]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
 
-- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries)
+- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries).
 
 ## Configuration
 
 Hornet uses several JSON configuration files that can be adjusted based on your deployment and use cases:
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/getting_started.md]:
```suggestion
Hornet uses two JSON configuration files that you can tweak based on your deployment requirements:
```

File: documentation/docs/getting_started/nodes_101.md [hunk 8]
```diff
@@ -1,25 +1,27 @@
...
+- explanation
+
 ---
 
 # Nodes 101
 
-The IOTA network is a distributed type of network called Tangle.  The network is distributed among plenty of servers
-called nodes. Nodes are the backbone of an IOTA network. This section covers what nodes do in an IOTA network.
+The IOTA network is a distributed type of network called Tangle. The network is distributed among several servers
+called nodes which are the backbone of an IOTA network. This article explains what nodes do in an IOTA network.
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/nodes_101.md]:
```suggestion
called nodes which are the backbone of an IOTA network.
```
The last sentence seems a bit superfluous.

File: documentation/docs/getting_started/security_101.md [hunk 9]
```diff
@@ -1,52 +1,63 @@
...
-When you're configuring your node, you have the option to allow it to do proof of work. If you enable this feature, clients can ask your node to do remote proof of work.
 
-Proof of work takes time and uses your node's computational power. So, you should consider enabling it according to your infrastructure.
+When you are configuring your node, you have the option to allow it to do proof of work (PoW). If you enable this feature, clients can ask your node to remotely do PoW.
+
+PoW takes time and uses your node's computational power. So, consider enabling it according to your infrastructure.
 
 ## Load Balancing
+
 If you run more than one node, it's a good practice to make sure that you distribute the API requests among all of them.
```
Comment (lucas-tortora) [inline documentation/docs/getting_started/security_101.md]:
```suggestion
If you run more than one node, it is a good practice to make sure that you distribute the API requests among all of them.
```

**Review by aleksei-korolev — APPROVED** [review]

File: documentation/docs/getting_started/getting_started.md (hunk 7 above)
Comment (charlesthompson3) [reply documentation/docs/getting_started/getting_started.md]: I added "several" as a cover in case there were more since the original doc was published👌

**Comment by charlesthompson3** [comment]
@aleksei-korolev and @lucas-tortora, accidentally pushed more updates to this branch but...it's all technically in the same section of the wiki🤷.

In the files changed, you can stop as soon as you get to nodes 101.

File: documentation/docs/getting_started/bootstrap_from_a_genesis_snapshot.md [hunk 10]
```diff
@@ -1,12 +1,13 @@
 ---
+description: Please follow these instructions to bootstrap the Chrysalis Phase 2 Hornet node from the Genesis Snapshot.
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/bootstrap_from_a_genesis_snapshot.md]:
```suggestion
description: How to bootstrap the Chrysalis Phase 2 Hornet node from the Genesis Snapshot.
```

File: documentation/docs/getting_started/getting_started.md [hunk 11]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
-Recommended installation methods:
+You can install Hornet on Linux using:
 
-- [Hornet apt repository](hornet_apt_repository.md#hornet-apt-repository-linux-distro-specific)
-- [Docker image](using_docker.md)
+- [The hornet apt repository](https://wiki.iota.org/hornet/getting_started/hornet_apt_repository).
+- [The docker image](https://wiki.iota.org/hornet/getting_started/using_docker).
 
-Other installation methods:
+Alternatively, you can buuld Hornet by following these instructions:
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/getting_started.md]: buuld

File: documentation/docs/getting_started/getting_started.md [hunk 12]
```diff
@@ -64,42 +68,42 @@ Hornet ships as a single executable binary (`hornet` or `hornet.exe`) and some J
...
 
-- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries)
-- [Build from source](hornet_apt_repository.md#build-from-source)
+- [Prebuilt binary files](hornet_apt_repository.md#pre-built-binaries), or
+- [Built from the source](hornet_apt_repository.md#build-from-source).
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/getting_started.md]: The original is correct (build it from the source?).

File: documentation/docs/getting_started/security_101.md [hunk 13]
```diff
@@ -1,52 +1,63 @@
...
+
+You can take measures to protect your device from unauthorized access when logging in through SSH from several readily avilable sources. The [10 Steps to Secure Open SSH](https://blog.devolutions.net/2017/04/10-steps-to-secure-open-ssh) guide and the tools found on [Fail2ban](https://www.fail2ban.org/wiki/index.php/Main_Page) should help strengthen your node's security.
 
 ### Blocking Unnecessary Ports
-Attackers can abuse any open ports on your device. To secure your device against attacks on unused open ports, you should close all ports except those that are in use.
 
-You can use a firewall to accomplish this. All operating systems include firewall options. By having a firewall in place, you can completely block unused and unnecessary ports.
+Attackers can abuse any open ports on your device. To secure your device against attacks on unused open ports, you can close all ports except ones you are using.
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/security_101.md]:
```suggestion
Attackers can abuse any open ports on your device. To secure your device against attacks on unused open ports, you can close all ports except the ones that you are using.
```

File: documentation/docs/getting_started/using_docker.md [hunk 14]
```diff
@@ -7,40 +9,42 @@ keywords:
...
 
 ## Prepare
 
 1. If you want to use alternative ports, edit the `config.json` file.
 
-2. Add your neighbors addressed to the `peering.json` file.
+2. Add your neighbors addresses to the `peering.json` file.
 
-The Docker image runs under user with user id 65532 and group id 65532. To make sure there are no permission issues you will need to:
+The Docker image runs under "user" with user id 65532 and group id 65532. To make sure there are no permission issues, you will need to:
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/using_docker.md]: why quotation marks?

File: documentation/docs/getting_started/using_docker.md [hunk 15]
```diff
@@ -95,16 +99,16 @@ docker run \
...
 * `--restart always` Instructs Docker to restart the container after Docker reboots.
 * `--name hornet` Name of the running container instance. You can refer to the given container by this name.
-* `--net=host` Instructs Docker to use the host's network, so the network is not isolated. We recommend that you run on host network for better performance.  This way, the container will also open any ports it needs on the host network, so you will not need to specify any ports.
+* `--net=host` Instructs Docker to use the host's network, so the network is not isolated. We recommend that you run on host network for better performance. This way, the container will also open any ports it needs on the host network, so you will not need to specify any ports.
 * `--ulimit nofile=8192:8192` increases the ulimits inside the container. This is important when running with large databases.
 * `-d` Instructs Docker to run the container instance in a detached mode (daemon).
 
 
-You can run `docker stop -t 300 hornet` to gracefully end the process.
+You can run `docker stop -t 300 hornet` to end the process.
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/using_docker.md]: This is actually a technical term, albeit jargo. Ask devs for details.

File: documentation/docs/getting_started/using_docker_compose.md [hunk 16]
```diff
@@ -1,24 +1,25 @@
...
 
 # Using Docker Compose
 
-Make sure that you have read [Using Docker](using_docker.md) before you continue reading as this chapter is for advanced users.  You can find more information on Docker Compose in the [official Docker Compose documentation](https://docs.docker.com/compose/).
+Make sure that you have read [Using Docker](using_docker.md) before you continue reading as this article is for advanced users. You can find more information on Docker Compose in the [official Docker Compose documentation](https://docs.docker.com/compose/).
 
 ## Using Your Own Docker Compose File For Running Hornet
 
-Docker Compose is a tool on top of the Docker engine that enables you to define Docker parameters in a structured way using a `docker-compose.yaml` file. You can create and start the container with a single `docker-compose` command based on your configuration.
+Docker Compose is a tool on top of the Docker engine that help you to define Docker parameters in a structured way using a `docker-compose.yaml` file. You can create and start the container with a single `docker-compose` command based on your configuration.
```
Comment (aleksei-korolev) [inline documentation/docs/getting_started/using_docker_compose.md:20]:
```suggestion
Docker Compose is a tool on top of the Docker engine that helps you to define Docker parameters in a structured way using a `docker-compose.yaml` file. You can create and start the container with a single `docker-compose` command based on your configuration.
```

**Review by aleksei-korolev — CHANGES_REQUESTED** [review]

File: documentation/docs/getting_started/using_docker.md (hunk 14 above)
Comment (charlesthompson3) [reply documentation/docs/getting_started/using_docker.md]: So they wouldn't be confused it was referring to _them_ as a user. But I'll swap it back.

File: documentation/docs/getting_started/using_docker.md (hunk 15 above)
Comment (charlesthompson3) [reply documentation/docs/getting_started/using_docker.md]: 👍

**Comment by muXxer** [comment]
Changes will be merged with #1376

**Pull request closed without merging**